src/jit/compiler.h

```diff
@@ -3407,6 +3407,10 @@ class Compiler
     NamedIntrinsic lookupNamedIntrinsic(CORINFO_METHOD_HANDLE method);
 
 #ifdef FEATURE_HW_INTRINSICS
+    GenTree* impVectorHelperIntrinsic(NamedIntrinsic        intrinsic,
```

Comment:
Shall we unify the name to `Base` or `Vector`?

Reply:
I'll normalize to `Base`